src/index.js

```diff
@@ -1,15 +1,12 @@
 import './style.css';
 
+import { saveTasks, savedTasks } from './modules/edit.js';
+
 const taskInput = document.getElementById('input');
 const addButton = document.getElementById('button');
 const todoList = document.getElementById('todo-list');
-const completeButton = document.querySelector('.complete');
-
-const savedTasks = JSON.parse(localStorage.getItem('tasks')) || [];
 
-function saveTasks() {
-  localStorage.setItem('tasks', JSON.stringify(savedTasks));
-}
+// const completeButton = document.querySelector('.complete');
```

Comment:
Great job @SAZZAD1Q2 🤝

- [ ] [Optional] Kindly remove all the commented code from your project as this is regarded as a bad practice.